i18n/language/language.go

```diff
@@ -17,23 +19,35 @@ type Language struct {
 	*PluralSpec
 }
 
+// NewLanguage returns new Language with specified tag and spec.
+func NewLanguage(tag string, spec *PluralSpec) *Language {
```

Comment:
Is this only a problem inside of the package? If these types of errors happen outside of the package then this isn't a backwards compatible change.